std/special/compiler_rt/extendXfYf2.zig

```diff
@@ -10,6 +10,14 @@ pub extern fn __extendsftf2(a: f32) f128 {
     return extendXfYf2(f128, f32, a);
 }
 
+pub extern fn __extendhfsf2(a: u16) f32 {
+    return extendXfYf2(f32, f16, @bitCast(f16, a));
+}
+
+pub extern fn __gnu_h2f_ieee(a: u16) f32 {
```

Comment:
I think this would be better as an actual symbol alias like this:

https://github.com/ziglang/zig/blob/8e714289cac55fd6f793cf21cea5fa1930edb985/std/special/compiler_rt/index.zig#L23-L24